src/linear_algebra/dimension.lean

```diff
@@ -149,16 +153,22 @@ lemma dim_span_set {s : set V} (hs : linear_independent K (λ x, x : s → V)) :
   dim K ↥(span K s) = cardinal.mk s :=
 by { rw [← @set_of_mem_eq _ s, ← subtype.range_coe_subtype], exact dim_span hs }
 
-lemma cardinal_le_dim_of_linear_independent
-  {ι : Type u'} {v : ι → V} (hv : linear_independent K v) :
-  (cardinal.mk ι) ≤ (dim.{u u'} K V) :=
+lemma {v} cardinal_lift_le_dim_of_linear_independent
```

Comment:
Do you really want to have an explicit universe level `v` and a function `v` in the same statement?

Reply:
Apparently lean is fine with it. I guess they live in different namespaces?

Reply:
Sure, Lean is not confused, I was more afraid readers might be. Thanks for the change.